README.md

```diff
@@ -5,6 +5,16 @@ Code for performing real-time shimming using external MRI shim coils
 
 ## Installation
 
+The first dependency you need is [`dcm2niix`](https://github.com/rordenlab/dcm2niix) >= v1.0.20200331.
+You can download precompiled binaries from https://github.com/rordenlab/dcm2niix/releases/latest -- make
+sure to add them to your PATH.
+It is also [in the AUR](https://aur.archlinux.org/packages/dcm2niix/) (`pikaur -Sy dcm2niix`),
+[`brew`](https://github.com/Homebrew/homebrew-core/blob/master/Formula/dcm2niix.rb) (`brew install dcm2niix`),
+and [`conda`](https://anaconda.org/conda-forge/dcm2niix) (`conda install -c conda-forge/label/cf202003 dcm2niix`).
+Unfortunately, the version [currently in Debian/Ubuntu](https://packages.ubuntu.com/eoan/dcm2niix) (`apt install dcm2niix`) is too old to work reliably.
```

Comment:
the problem with that, is we won't be testing those other installation methods. 

to prevent any issue in the future like https://github.com/shimming-toolbox/shimming-toolbox-py/pull/18#issuecomment-665818569, i would suggest to instruct users to only fetch the latest binaries provided on github. Alternatively, we could implement an installer which will fetch the appropriate binary depending on the platform.

Reply:
An integrated installer won't integrate well with external systems or distros. It would mean running everything in an expensively isolated box.

Reply:
ok, in that case, we should have a routine in shimmingtoolbox that verifies which version of dcm2niix is run by default by the user, and if this is an incorrect version, it errors and quit-- how about that?